frontend/src/components/elements/ArrowDataFrame/DataFrameCell.test.tsx

```diff
@@ -40,18 +43,32 @@ const getProps = (
 describe("DataFrameCell Element", () => {
   it("renders without crashing", () => {
     const props = getProps()
-    const wrapper = shallow(<DataFrameCell {...props} />)
+    const wrapper = mount(
+      <ThemeProvider theme={lightTheme.emotion} baseuiTheme={lightBaseUITheme}>
+        <DataFrameCell {...props} />
+      </ThemeProvider>
+    )
```

Comment:
I believe we have a `mount` function exported from `test_util.tsx` that lets you pass a theme to it so that having to wrap things in a `ThemeProvider` yourself doesn't get so tedious